build.sc

```diff
@@ -7,25 +7,24 @@ import scalalib._
 // support BSP
 import mill.bsp._
 
-object %NAME% extends SbtModule { m =>
+object foo extends SbtModule { m =>
```

Comment:
Is this change intended?

Reply:
No it was not, whoops, I made that change for local testing.

Thanks for pointing it out, will fix.

Reply:
Fixed, thanks @jiegec!